t3/api/proto/external/v1/external.proto

```diff
@@ -0,0 +1,309 @@
+/**
+ * DO NOT MODIFY THIS PROTOBUF
+ *
+ * This has been copied directly from the link below. This should likely never
+ * be modified in this repo, and instead will eventually be replaced by an
+ * externally published and versioned protobuf module. The only change is in
+ * the definition of the package name, which must be suffixed with a version
+ * number.
+ *
+ * https://github.com/mobilecoinofficial/android-sdk/blob/336b2efe4889ef21cfbc4e7e451b83aa1c13142a/android-sdk/src/main/proto/external.proto
```

Comment:
What does the android-sdk has to do with this repo? Why is this not copied from the main repo?

Reply:
No idea, this is what I found on buf.build